packages/chrome/src/elements/sheet/Sheet.spec.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useRef, useState, forwardRef } from 'react';
+import userEvent from '@testing-library/user-event';
+import { render } from 'garden-test-utils';
+import { ThemeProvider, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+import { Sheet, ISheetProps } from './Sheet';
+
+describe('Sheet', () => {
+  const Example = forwardRef<HTMLElement, ISheetProps>((props: ISheetProps, ref) => {
+    const [isOpen, setIsOpen] = useState(() => props.isOpen);
+    const buttonRef = useRef<HTMLButtonElement>(null);
+
+    return (
+      <>
+        <button ref={buttonRef} onClick={() => setIsOpen(!isOpen)}>
+          Toggle Sheet
+        </button>
+        <Sheet ref={ref} isOpen={isOpen} {...props}>
+          <Sheet.Header data-test-id="header">
+            <Sheet.Title data-test-id="title">title</Sheet.Title>
+            <Sheet.Description data-test-id="description">description</Sheet.Description>
+          </Sheet.Header>
+          <Sheet.Body>body</Sheet.Body>
+          <Sheet.Footer data-test-id="footer">
+            <Sheet.FooterItem>
+              <button>button1</button>
+            </Sheet.FooterItem>
+            <Sheet.FooterItem>
+              <button>button2</button>
+            </Sheet.FooterItem>
+          </Sheet.Footer>
+          <Sheet.Close data-test-id="close-btn" />
```

Comment:
I think these test IDs can be removed in favor of [priority querying](https://testing-library.com/docs/queries/about/#priority). For elements like `Sheet.Close`, simulation of `tab` + `enter` key events can be used to make assertions against trigger behavior.

Reply:
Agreed, I've updated and removed the test ids.